Tests/Unit/InMemoryAccessTokenManagerTest.php

```diff
@@ -20,52 +20,128 @@ public function testClearExpired(): void
         timecop_freeze(new DateTimeImmutable());
 
         try {
-            $testData = $this->buildClearExpiredTestData();
+            $testData = $this->buildTestData(
```

Comment:
@franjo-zadelj-trikoder This still uses the `buildTestData` stuff which we've agreed that should be removed, but it is still present here.

Reply:
I missed all of those in-memory test when reverting, it's ok now